### DISCUSSION TITLE
Where 'Remote Projects' recents are located on Mac?

### DISCUSSION THREAD

**Discussion by vdimir** [zed-industries/zed#17535] [Feature Requests]
Hi there! Thanks so much for an excellent product! ❤️ 

My question is related to removing invalid entries in Remote Projects. The loading gets stuck because I accidentally opened a huge folder, and I manually killed the server on the remote, which resulted in a broken entry, that I cannot remove, because I keep getting errors when trying to delete the folder or the remote server:

<details>
<summary>Details</summary>

<img width="250" alt="Screenshot 2024-09-07 at 15 11 46" src="https://github.com/user-attachments/assets/4b1cae33-a5ea-4671-8425-b01a6af15986">
<img width="245" alt="Screenshot 2024-09-07 at 15 11 21" src="https://github.com/user-attachments/assets/30093ea8-2a38-4e23-924b-aa437f2f111c">

</details>

I cleaned up `~/Library/Application\ Support/Zed` and `~/.cache/zed`, and reinstalled the app, but I still see non-empty `Remote Projects` list (although recent projects have been cleared).

**Comment by Moshyfawn** [comment]
The remote server entries are stored in `~/Library/Application Support/Zed/db/<zed-instance-name>` in the `dev_server_projects` table. The easiest way to deal with this is to delete the entire `~/Library/Application Support/Zed/db` folder.

**Reply by vdimir** [reply]
For some reason cleaning up `~/Library/Application Support/Zed/db` doesn't work:

<details>
<summary>Details</summary>


https://github.com/user-attachments/assets/bcf2d97c-2920-4096-85a4-96a6b29a6b43

</details>


In root `Library` folder `/Library/Application Support/Zed` doesn't exists

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
